### ISSUE TITLE
We need more themes

### ISSUE THREAD

**Issue by docwilmot** [backdrop/backdrop-issues#982]
A strange issue to write, but considering a recent discussion to expose Backdrop more to the world, it occurs to me that what website seekers would look at first and the most is what a good website **looks** like, not its internals.

We have one theme that was probably my first Backdrop project (Cleanish theme) and a couple by @biolithic which are pretty much works in progress. 

Furthermore, after thinking about this, I considered working on a theme or two, which I still might, but the porting now requires a split between layouts and themes which I still havent quite reasoned out. [See a thread](http://www.reddit.com/r/backdrop/comments/2zq6x5/reconciling_the_ux_of_layouts_and_themes/) on our neglected Reddit. 

In short though I think the lack of themes is a significant enough issue to post here, if at least to motivate the themers amongst us to work on it.

**Comment by MrHaroldA** [comment]
I'd suggest not to add any themes to Backdrop core, but instead browse through a remote "theme repository" or something like that.

You can never add the specific theme a user is looking for ...

**Comment by klonos** [comment]
@MrHaroldA: what @docwilmot is talking about is not including more themes with core but the fact that we do not have enough themes in contrib (in contrary to the big number of contrib modules that were ported from Drupal).

As for the theme repository, you'd be happy to know that we are already working on a "Themes" section for the main site Backdrop.org (as well as a "Modules" section and a "Layouts" section):

[Build project listings on backdropcms.org](https://github.com/backdrop-ops/backdropcms.org/issues/65)

More exciting is that we are also working on an "internal"/"remote" browser for projects (be it themes, modules or layouts) that will work from within each Backdrop installation so that site administrators can "automagically" browse for available projects, download and install them without ever having to leave their website administration pages:

[Add browser server to BackdropCMS.org](https://github.com/backdrop-ops/backdropcms.org/issues/73) (the server part that needs to be implemented in the Backdrop.org side)

[Create a project browser for modules, themes, layouts etc](https://github.com/backdrop/backdrop-issues/issues/870) (the UI part/module that is to be implemented in Backdrop CMS - hopefully in version 1.2.0)

Hope that makes it all clear.

**Comment by biolithic** [comment]
Hi @docwilmot,

Yes, there is an issue here.  But it is not wise to just "port the top ten
themes on Drupal", because:

1) "we might be going after a Wordpress market"  -- I was talking to two
business owners over the weekend who use Wordpress.  They have not heard of
Zen theme or Omega or Maranelli theme.  Should we teach them the Drupal
theming ways through Backdrop, or find our own voice?

2) "zen grids!  960gs!  singularity!" -- layouts already do the grid
system...there is not a real reason to 100% copy paste Drupal grid CSS
themes when we already have Layouts.

3) we already have a good admin theme, your clean theme, several Bootstrap
themes, Aurora/Wormhole, people can join in on helping the starter kits of
Zurb Foundation, Android/Material UI, or Yahoo/PureCSS reach maturity.
Where "we need an admin theme like Rubik" for Drupal...not so much for
Backdrop.  Where "we need a responsive theme" for Drupal...not so much for
Backdrop.  So there is less need out of the box because Backdrop already
solves several organizational needs with the attention to UI.

4) Because the Drupal shops I've worked in used Zen or Omega or Aurora
(very minima themes) for every single site produced, and end users think
they like fully formed themes like Bootstrap or Xtheme or maximal themes,
it remains to be seen who is the market and what do they want.    We could
use Backdrop sites to give us ideas of where there is a product-market
fit.  The Mothership theme is very exciting for a coder ("no code to mess
me up!"), while Zurb is exciting for the marketer ("lots of code I can
reuse!"), but neither cares about the other.

5) The market will dictate the types of themes as well.  What do I want in
a theme?  Grunt + libSASS + NodeJS reload...basically there is no time at
all between thought and see the result as you type valid CSS. What does an
end user want?  E-Z CSS + WYSIWIG + widgets like modals and the big
slideshow on the home page.  So there is a disparity in who gets what into
a theme.

6) Are some widgets modules or theme additions?  Hamburger menus, parallax
scrolling/single page sites, slideshows, AJAXy add more buttons and the
like are currently Backdrop modules, whereas Wordpress themes advertise
these things as part of the theme and the reason you need to use the
theme.  So, there might be a less need for mass amounts of themes if you
match functionality up with modules and general pretty-ness up with themes.

6b) Since we are using a newer version of JQuery and Libraries module was
up in the air, nobody has been really sure on whether plugins or libraries
that relied on Jquery or CSS animations were a module, a library/plugin, or
part of a theme.  We're all busy, sorry!

7) With the project browser marching ahead (themes) and possibly the
screenshot browsing instead of text/search browsing, will it be more
important to build more niche themes with a screenshot of a website with
the theme, or a more general purpose blank theme?  For example,

"I'm an environmental agency.  I want a "green" looking theme.  Through
project browser, I see a screenshot of a green website and push the button
to download.  I choose it because through browsing, I see a nice green
website.  Now I want the developer to make it using this green theme"
 VS
 "I'm a developer and I do a text search for HTML5 clean base theme.  I
download it because I can code it to be shades of green for the client, an
environmental agency".

This above story may sound stupid, but it might have some truth to it.  It
is possible that we used text searches on Drupal.org to find things in the
past, but new users may not go to Backdropcms.org to search based on
keywords because of project browser in their site.

"I just got into Backdrop and it is cool.  You download it, and then to add
stuff to your website you just look through the thumbnails in the admin
part and push the screenshot you want and it gets added to your website."

 -- it is a possibility that this + Drush/Brush might be the ways of
accessing themes, and if we make a hundred starter themes, no one will
download them because they don't have cool screenshots and pros already
have Drush/Brush and their go-to theme anyways.  Why "push the
screenshot"?  We use a desktop computer and a mouse and type in search
queries.  It is possible that through project browser and pantheon decision
makers might use their phone/tablet to try on different themes using their
mobile devices by browsing/scrolling/tapping rather than going to the
mother site on their computer, typing in "responsive business theme",
clicking clicking FTP'ing etc....  This might not be but it might.  I
didn't tell anyone to work on project browser, but people sure work on
project browser!


The above are just my personal hypothesis after fixing websites for
marketing agencies, law firms, townships, etc...

I was a little interrupted recently by family from out of town dropping in
and my birthday and a local hackathon I had on my calendar for a long time
(and I still work fixing or adding to existing sites, not new Backdrop
sites or Drupal8 sites), but will get back to it shortly.
If you are interested, after that busyness is over, I am just starting a
blog series on a Backdrop site build (how-to of fictional site)
http://www.andymartha.com/backdrop_tutorial1/backdrop-cms-theming-tutorial-v11
.  If anyone is interested in a guest blog, I am open to it. And then
afterwards
http://www.andymartha.com/backdrop_tutorial1/foundation5-backdrop-cms-tutorial
for how 3rd-party existing CSS frameworks (knowledge transfer) fit into
Backdrop.  Do you build everything from scratch because of page bloat?
Great, don't tune in, this is more to get interest from non-developers.
 ("Why do you write JQuery instead of ECMAscript when real programmers
write all teh codez?", "Keep all of our employees on the same page,
employee turnover/ramp up time, large documentation, time, habit, etc..")



Thanks for noticing!

**Comment by docwilmot** [comment]
Made one more https://github.com/backdrop-contrib/colihaut, please test.

**Comment by biolithic** [comment]
good job, will check it out tonight!

**Comment by docwilmot** [comment]
Another, clone of WP default theme 'twentyfifteen'. https://github.com/backdrop-contrib/mero
No screenshot yet, and not yet mobile ready, but looks good I think. Three color module settings. Would be grateful if you could lease test and comment.

**Comment by docwilmot** [comment]
See http://wilmurt.com/ for demo

**Comment by quicksketch** [comment]
Hey @docwilmot, I think we're good to close this issue. We can still use more themes (of course), but as a call to action, do you think we've made some headway here? Reopen if you think there's still more to do here.
